docs/docs/index.md

```diff
@@ -7,9 +7,8 @@ Mathesar is a straightforward open source tool that provides a **spreadsheet-lik
 You can use Mathesar to build **data models**, **enter data**, and even **build reports**.
 
 ## Installation
-You host Mathesar on your own server. Currently, we only support installing using Docker Compose, although we will support additional installation methods soon.
 
-- [Install with Docker Compose](installation-dc/quickstart.md).
+[Install Mathesar](install/index.md)
```

Comment:
`installation-dc` was a deliberate URL choice. We're going to have other installation methods soon, and we'll have other "installation" folders.

Also, if we change the URLs now, then we'll need to do a new release to update the install script to point at the right documentation.

Reply:
Ugh. Ok. I'd like to try setting up redirects to deal with the the broken link issue.

Reply:
@kgodey

> We're going to have other installation methods soon, and we'll have other "installation" folders.

This is actually the motivation for some of my work in this PR. I think it's a cleaner now. The page you point to in the diff `/install` is a landing page for all installations, regardless of deployment type. That's where I'd like to eventually direct people from many places, such as the GH README, the marketing site and so on. The docs homepage also directs there. The `/install` landing page is very sparse at the moment. Really it's just one additional hoop to jump through, which is not ideal in the short term. But eventually it will serve as a fork in the road where we provide more guidance on choosing the appropriate deployment type. Even though this PR adds that additional "hoop", I still think the install flow from start to finish is much clearer and easy to follow.

> if we change the URLs now, then we'll need to do a new release to update the install script to point at the right documentation.

I pushed f7599c43baf082cfd357a73cfc9d1a845068d921 which adds redirects so that there shouldn't be any broken links.

Reply:
@seancolsen That makes sense, I didn't pay attention to this PR closely enough the first time, I thought everything was being renamed.

My concerns are satisfied. I'll leave it to @mathemancer to review and merge.